packages/gestalt/src/Module/ExpandableItem.js

```diff
@@ -85,7 +85,7 @@ export default function ModuleExpandableItem({
             </Box>
 
             {Boolean(children) && (
-              <Box id={id} padding={1}>
+              <Box id={id} padding={1} display="flex" alignItems="center" maxHeight={24}>
```

Comment:
can you add comment that maxHeight={24} is because of the helperButton?

Reply:
done